site/app/templates/polls/AllPollsPageInstructor.twig

```diff
@@ -29,31 +34,12 @@
                 {% for poll in todays_polls %}
                     <tr>
                         <td> {{ poll.getName() }} </td>
-                        {% if poll.isOpen() %}
-                            <td>
-                                <form method="post" action="{{ base_url }}/setEnded">
-                                    <input type="hidden" name="csrf_token" value="{{ csrf_token }}"/>
-                                    <input type="hidden" name="poll_id" value="{{ poll.getID() }}"/>
-                                    <button type="submit" class="btn btn-primary">End Poll</button>
-                                </form>
-                            </td>
-                        {% elseif poll.isClosed() %}
-                            <td>
-                                <form method="post" action="{{ base_url }}/setOpen">
-                                    <input type="hidden" name="csrf_token" value="{{ csrf_token }}"/>
-                                    <input type="hidden" name="poll_id" value="{{ poll.getID() }}"/>
-                                    <button type="submit" class="btn btn-primary">Open Poll</button>
-                                </form>
-                            </td>
-                        {% elseif poll.isEnded() %}
-                            <td>
-                                <form method="post" action="{{ base_url }}/setOpen">
-                                    <input type="hidden" name="csrf_token" value="{{ csrf_token }}"/>
-                                    <input type="hidden" name="poll_id" value="{{ poll.getID() }}"/>
-                                    <button type="submit" class="btn btn-primary">Reopen Poll</button>
-                                </form>
-                            </td>
-                        {% endif %}
+                        <td>
+                            <input class="key_to_click" tabindex="0" type="checkbox" id="poll_{{ poll.getID() }}_visible" aria-label="label" onclick='updatePollVisible("{{ poll.getID() }}")' {{ (poll.isOpen() or poll.isEnded) ? 'checked' : '' }}>
+                        </td>
+                        <td>
+                            <input class="key_to_click" tabindex="1" type="checkbox" id="poll_{{ poll.getID() }}_view_results" aria-label="label" onclick='updatePollAcceptingAnswers("{{ poll.getID() }}")' {{ poll.isOpen ? 'checked' : ''}}>
```

Comment:
Same comment as above and remove the tabindex, we should default to the natural input order in this case since the inputs are nested within a table.
A tabindex of 0 would give the element a tab order based on its location in the DOM, a tab index of 1 would hard code it as the first element to be selected when hitting tab. This causes the user to jump to the "Accepting Answers" input even though they should start at the input before.

Reply:
Oh sorry I meant you can remove both tabindexes and the browsers default ordering will work here.

I meant to say tabindex 1 makes it so when the user presses tab they will be brought to that element which is not what we want since there are other element's on the page that should be visited first (like the skip to main content section)

Reply:
Okay, thanks for the knowledge! I removed both tab indexes then.